### ISSUE TITLE
Bug Report: Backgrounds

### ISSUE THREAD

**Issue by Toppat-dalek** [KadeArchive/Kade-Engine#344]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.
Windows (`x86_64`)
## What version of the game are you using? Look in the bottom left corner of the main menu.
1.4.2 Kade Dev
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
Load spookeez in week 1
## Please describe your issue. Provide extensive detail and images if possible.
whenever you load a background in the week it's not expected to be in the background just turns black. EX: spooky background in week 1,
![f](https://user-images.githubusercontent.com/77013267/116732812-3d742d80-a9b9-11eb-930b-0bb9f109892a.PNG)

**Comment by theDetourist** [comment]
Yeah, it's part of #341, the game preloads based on the week you are. Fixing that would required a major overhaul of the loading system that isn't worth doing right now until Week 7 drops on Github.

**Comment by RagdollClash** [comment]
fixed for desktop with d707b0223be7e8e532b5b824b02e388fa41b74ff, but still an issue for html5

https://user-images.githubusercontent.com/15317421/116757044-1ff79180-a9c2-11eb-90c7-60aa955df79a.mp4

**Comment by haya3218** [comment]
The second preload parameter doesnt work on html5 for some reason 🤔

**Comment by raf13lol** [comment]
close this lol
